docs/src/inscriptions/recursion.md

```diff
@@ -48,6 +48,17 @@ plain-text responses.
 - `/blockhash/<HEIGHT>`: block hash at given block height.
 - `/blocktime`: UNIX time stamp of latest block.
 
+### Self Referencing
+
+An inscription may fetch data about itself using the endpoints above by 
+using its own inscription ID. Since all inscriptions are rendered with 
+a URL of `/content/<INSCRIPTION_ID>`, it can retrieve its own inscription 
+ID from `window.location.pathname` using the following line (or similar)
+
+```js
+const inscriptionId = window.location.pathname.match(/\/content\/([\da-f]{64}i\d+)/i)[1]
+```
+
```

Comment:
I don't think we need any of this. We shouldn't enforce how people use the window object or even give them regex none of the docs have any examples of inscription implementation.

Reply:
Yeah, I'm fine with removing this as it's not strictly necessary. However, I don't think it's bad to include examples in documentation. Heaps of software documentation includes examples, they're designed to make developers lives easier, so I don't see the issue with it. This was meant to just be an example of how you could do it, not enforcing people to do it this way, as it is a common question that new ordinals developers have. 

Maybe it belongs somewhere in the guides section instead, but like I said, this isn't necessary, so I have removed it for now